### ISSUE TITLE
Make cartesian axis endpoint drag elements visible to the end users

### ISSUE THREAD

**Issue by archmoj** [plotly/plotly.js#5116]
There is a nice feature built into `cartesian` axes in which one could interact with the graph by dragging the endpoints.
Unfortunately at the moment this is rather hidden to most users until someone notices the cursor change and accidentally drags the invisible element!
So it would be nice to have the option of showing the location of these drag elements by a line or something.

**Comment by alexcjohnson** [comment]
I wonder if we could have these regions glow a little when you hover over them? Just enough to let folks know there's something active there. Note that it's not just the axis endpoints, it's the whole middle of each axis too, and distinct behavior in the corners.

**Comment by nicolaskruchten** [comment]
Don't we have the cursor change already? If not that's a good start just when hovering maybe

**Comment by alexcjohnson** [comment]
yes, the cursor changes - but that's pretty subtle, might be nice to find something slightly more obvious we can do.

**Comment by nicolaskruchten** [comment]
See also #2942
